src/layouts/SiteDashboard/components/ReviewRequestCard.tsx

```diff
@@ -82,9 +82,13 @@ export const ReviewRequestCard = ({
         >
           <Text textStyle="subhead-1">
             View <b>{reviewRequest.changedFiles}</b> changed{" "}
-            {reviewRequest.changedFiles === 1 ? "item" : "items"},{" "}
-            <b>{reviewRequest.newComments}</b> new{" "}
-            {reviewRequest.newComments === 1 ? "comment" : "comments"}
+            {reviewRequest.changedFiles === 1 ? "item" : "items"}
+            {reviewRequest.newComments > 0 && (
+              <>
+                , <b>{reviewRequest.newComments}</b> new{" "}
+                {reviewRequest.newComments === 1 ? "comment" : "comments"}
+              </>
+            )}
```

Comment:
wdyt about refactoring this so that the whole chunk is clear? can't really tell the message HAHAHA

Reply:
I abstracted out the singular plural part as a separate function, together with the formatting for the bold in cd2f64a.